### ISSUE TITLE
Turkish letters

### ISSUE THREAD

**Issue by CoskunSunali** [antijingoist/open-dyslexic#69]
Hi!

First of all, great job!

I read somewhere that you might add additional letters on demand.

Is it possible that you add the following letters?

Ğ (uppercase)
ğ (lowercase)
İ (uppercase)
Ö (uppercase)
Ü (uppercase)

https://www.dafont.com/open-dyslexic.font?text=%26%23286%3B+%26%23287%3B+%26%23304%3B+%D6+%FC

Your work is really appreciated...

**Comment by antijingoist** [comment]
Yes. On-demand is kinda adhoc for a bit until I figure out a bit of a medical issue. I think I have 4/5 of these already in the newest GitHub version though. Can you let me know if I'm missing them still (I may have it, but have it in a character code that's unexpected, etc)?

**Comment by CoskunSunali** [comment]
Thanks for your reply. I am no expect when it comes to fonts and char codes. Would you mind letting me know how do I confirm if you have them on the GitHub version or not? I am eager to confirm/help.

**Comment by 1000i100** [comment]
i'm very interested in these : 
Ğ (uppercase)
ğ (lowercase)

It become a monetary symbol for a growing cryptomoney with human equality behaviors.
And in the humans, there are some dys ;)

In the dafont version, Ğ dont look good (but ǧ yes) and in the lastest master build, it look good for both.
